<?xml version='1.0' encoding='UTF-8'?>
<root version="2.3.8">
 <tabbed_widget name="Main Window" parent="main_window">
  <Tab tab_name="tab1" containers="1">
   <Container>
    <DockSplitter sizes="0.199773;0.199773;0.200908;0.199773;0.199773" orientation="-" count="5">
     <DockArea name="...">
      <plot mode="TimeSeries" style="Lines">
       <range left="128.677648" top="78.359636" right="148.659552" bottom="32.530367"/>
       <limitY/>
       <curve color="#eabd00" name="/iris_0/perf_analyzer/curr_net_cap/data"/>
       <curve color="#d62728" name="/crowds/target_net_capacity/data.0"/>
      </plot>
     </DockArea>
     <DockArea name="...">
      <plot mode="TimeSeries" style="Lines">
       <range left="128.701464" top="4.004692" right="148.626586" bottom="-2.369112"/>
       <limitY/>
       <curve color="#1f77b4" name="/scheduler/adf/data.0"/>
       <curve color="#04c2e5" name="/scheduler/scale_factor/data.0"/>
       <curve color="#1ac938" name="/scheduler/grad/data.0"/>
       <curve color="#ff7f0e" name="/iris_0/max_lin_acc/data"/>
      </plot>
     </DockArea>
     <DockArea name="...">
      <plot mode="TimeSeries" style="Lines">
       <range left="128.668434" top="26.354030" right="148.659961" bottom="-20.742872"/>
       <limitY/>
       <curve color="#1ac938" name="/iris_0/real_pose/orientation/pitch_deg"/>
       <curve color="#007c31" name="/iris_0/real_pose/orientation/roll_deg"/>
      </plot>
     </DockArea>
     <DockArea name="...">
      <plot mode="TimeSeries" style="Lines">
       <range left="128.701317" top="10.175000" right="148.626550" bottom="2.825000"/>
       <limitY/>
       <curve color="#1f77b4" name="/iris_0/max_lin_acc/data"/>
      </plot>
     </DockArea>
     <DockArea name="...">
      <plot mode="TimeSeries" style="Lines">
       <range left="128.749132" top="33.348283" right="148.659562" bottom="-0.799656"/>
       <limitY/>
       <curve color="#ff7188" name="/iris_0/perf_analyzer/instant_power/data"/>
      </plot>
     </DockArea>
    </DockSplitter>
   </Container>
  </Tab>
  <currentTabIndex index="0"/>
 </tabbed_widget>
 <use_relative_time_offset enabled="1"/>
 <!-- - - - - - - - - - - - - - - -->
 <!-- - - - - - - - - - - - - - - -->
 <Plugins>
  <plugin ID="DataLoad CSV">
   <default time_axis=""/>
  </plugin>
  <plugin ID="DataLoad ROS bags">
   <use_header_stamp value="false"/>
   <use_renaming_rules value="true"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
  </plugin>
  <plugin ID="DataLoad ULog"/>
  <plugin ID="MQTT Subscriber"/>
  <plugin ID="ROS Topic Subscriber">
   <use_header_stamp value="false"/>
   <use_renaming_rules value="true"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
  </plugin>
  <plugin ID="UDP Server"/>
  <plugin ID="WebSocket Server"/>
  <plugin ID="ZMQ Subscriber"/>
  <plugin ID="ROS /rosout Visualization" status="idle"/>
  <plugin ID="ROS Topic Re-Publisher" status="idle"/>
 </Plugins>
 <!-- - - - - - - - - - - - - - - -->
 <previouslyLoaded_Datafiles/>
 <previouslyLoaded_Streamer name="ROS Topic Subscriber"/>
 <!-- - - - - - - - - - - - - - - -->
 <customMathEquations/>
 <snippets/>
 <!-- - - - - - - - - - - - - - - -->
</root>

